packages/plugin-icp/src/utils/hex.ts

```diff
@@ -0,0 +1,21 @@
+// hex text -> number array
```

Comment:
We could consider moving this to the array utils as they are all related to arrays